src/common/Collision/Management/MMapManager.cpp

```diff
@@ -185,13 +185,9 @@ namespace MMAP
             LOG_DEBUG("maps", "MMAP:loadMap: Loaded mmtile %03i[%02i,%02i] into %03i[%02i,%02i]", mapId, x, y, mapId, header->x, header->y);
             return true;
         }
-        else
-        {
-            LOG_ERROR("maps", "MMAP:loadMap: Could not load %03u%02i%02i.mmtile into navmesh", mapId, x, y);
-            dtFree(data);
-            return false;
-        }
 
+        LOG_ERROR("maps", "MMAP:loadMap: Could not load %03u%02i%02i.mmtile into navmesh", mapId, x, y);
+        dtFree(data);
```

Comment:
Should this error always be sent?

Reply:
@Kitzunu this PR is not changing the behaviour, the `if` is returning so either you use `else` or not, the result is exactly the same

Reply:
![image](https://user-images.githubusercontent.com/75517/130639712-3b05fe20-29e6-4526-b825-4837ed4dae24.png)

Reply:
:/